rss/erc.xml

```diff
@@ -0,0 +1,33 @@
+---
+layout: null
+---
+<?xml version="1.0" encoding="UTF-8"?>
+<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
+  <channel>
+    <title>Ethereum EIPs - Last Call Review</title>
+    <description>All EIPs which are in the two-week "last call" status, please help review these and provide your feedback!</description>
+    <link>{{ site.url }}</link>
+    <atom:link href="{{ site.url }}/rss/last-call.xml" rel="self" type="application/rss+xml" />
+    <lastBuildDate>{{ site.time | date_to_rfc822 }}</lastBuildDate>
+    {% assign eips = site.pages | sort: 'eip' %}
+    {% for eip in eips %}
+      {% if eip.category == "ERC" %}
+      {% capture description %}
+        <p><strong>EIP #{{ eip.eip }} - {{eip.title }}</strong> is in Last Call status. It is authored by {{ eip.author }} and was originally created {{ eip.created }}. It is in the {{ eip.category }} category of type {{ eip.type }}. Please review and note any changes that should block acceptance.</p>
+        {% if eip.discussions-to %}
+          <p>The author has requested that discussions happen at the following URL: {{ eip.discussions-to }}</p>
```

Comment:
Can the discussion-to link be in an anchor tag?

Reply:
Thank you, fixed at a2534bfb77d80b6e4755f5ec0b8ba04b5d921cfb